extensions/amp-youtube/0.1/test/test-amp-youtube.js

```diff
@@ -72,6 +72,17 @@ describes.realWin('amp-youtube', {
     });
   });
 
```

Comment:
Also on the subject of tests. I'm wondering which tests is appropriate to replicate for `live-channelids`. If more of these should be tested with `live-channelids`, it would be a better approach to write a `runTestsForDatasource` function that encapsulates all tests applicable to both datasources and write something like: 

```
describe('with videoid', function() {
  runTests...({'data-videoid': 'myVideoId'});
}

describe('with live-channelid', function() {
  runTests...({'data-live-channelid': 'myChannelId'});
}
```

Reply:
It could make sense to run more tests for both types of video source to make sure that features we expect to behave the same for both actually behave the same. But things that we know will be different don't need to be run for both e.g. the `it('adds an img placeholder...')`